boards/shields/seeed_w5500/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Marcin Niestroj
```

Comment:
Nice to have: SPDX-FileCopyrightText:

Reply:
Thanks for suggestion. Haven't found any `SPDX-FileCopyrightText:` use in `boards/` directory yet. I would rather leave it as is for now just for consistency.